### ISSUE TITLE
3.0.2 release

### ISSUE THREAD

**Issue by da314pc** [raix/push#358]
@shivang007 For this release I think we should get the following issues updated:

1) https://github.com/raix/push/issues/341#issuecomment-434436787 

   I have found the fix, we need to investigate if we can update the build.gradle with the correct firebase 
   libraries

2) Test & Merge in these two pull request's 
https://github.com/raix/push/pull/327
https://github.com/raix/push/pull/354

3) Update the Readme: https://github.com/ToothlessGear/node-gcm already supports FCM, so there is confusion among some users

4) https://github.com/raix/push/issues/342

5) https://github.com/raix/push/issues/326

**Comment by shivang007** [comment]
@da314pc Regarding first point, meteor needs to make changes in the build process, that is not something I believe we (as a plugin) should touch, we can create an issue in the meteor and update our docs for the time being.

2. Yes let's do that, I will get in touch with the authors too and move this ahead. And create a pull request of my own if required.

3. Can you please provide some reference of those issues.

4. & 5. Will look into that.

Also, let's start closing the issues that are already fixed.

**Comment by shivang007** [comment]
Cool 👍

**Comment by pkumar-uk** [comment]
Regarding 1st Comment. I was able to make the notification work for Oreo using FCM without using build-extras.gradle. I suspect it works when you move to 1.8.0.2.

I am using phonegap-plugin-push@2.1.3.

**Comment by da314pc** [comment]
@pkumar-uk  did you use build-extras.gradle configuration:
https://github.com/meteor/meteor/issues/7600#issuecomment-383327945

@shivang007 I think for now maybe we just update the readme until we have tested the build extras build.

Everyone isn't on 1.8, im still using 1.6, But I will create a pull request probably within the next day to get the android channel changes.

**Comment by pkumar-uk** [comment]
> @pkumar-uk did you use build-extras.gradle configuration:

No, I did not use build-extras.gradle configuration as mentioned in meteor/meteor#7600 (comment)

**Comment by da314pc** [comment]
@shivang007 @aboire I have just created the Pull Request for the android channel Integration, I have added you as a reviewer. After that is merged in I will update the README docs on the instructions.

**Comment by shivang007** [comment]
@da314pc update the README docs in this request only, it will be easier to track.

**Comment by da314pc** [comment]
@shivang007 could you take a look at the update changes.

**Comment by shivang007** [comment]
@da314pc I was able to build the app by following the steps in your readme, so that solves the build problem. I will make sure to get an android 8 and test it by tomorrow. Meanwhile can you please make the other changes in the pull request, you have indented the file and hence it's showing extra code which is not going in this release. Just make sure that the files shows only those changes that you have made and we will move ahead with the merge.

**Comment by da314pc** [comment]
@shivang007 Thanks for helping getting the android 8 merged in. Now we can look at the FCM and new APN update pull request. Just let me know which ones you prefer to work on and I'll take a look at the other one

**Comment by shivang007** [comment]
@da314pc I can look at the FCM.

**Comment by paulincai** [comment]
Hi, you can find documentation and latest updates of everything in this repo: https://github.com/activitree/meteor-push this is up to date with all libraries and running in production. Perhaps this could help you to get the FCM up to date (which is not really the "standard") and you might want to use the Firebase-Admin instead. You might also want to consider to start pushing to that repo too, given the fact that it is already based on far more recent tech.
